### ISSUE TITLE
Notification settings not saved

### ISSUE THREAD

**Issue by torres01** [mchristopher/PokemonGo-DesktopMap#761]
It seems the notification settings works from time to time and remembers all the pokemons i would like to get a notification for. But today, after reopening the map for several times, it was blank again. I don't really see a pattern why it sometimes comes up with my saved pokemons and then suddenly works blank. Is it possible to reevaluate that option and fix this bug?

**Comment by Madman8** [comment]
I second that. It's very weird. Like when I created a new account... on the first log in, it had all my settings from the previous account, then I logged in with the previous account, and it was blank.

Out of no where, this afternoon, all my settings were there, but have since disappeared again.

No rhyme or reason, as I am doing the exact same thing each time. Click icon in dock, click login Google, and map opens. 

Super confused, but have given up on entering the pokemon for ignore and notify. It just gets to bee too long a list at this stage of the game.

Anyway, beggars can't be choosers. This is a BUTTLOAD better than the alternative!!!
